assets/src/dashboard/app/api/useStoryApi.js

```diff
@@ -194,32 +194,42 @@ const useStoryApi = (dataAdapter, { editStoryURL, storyApi }) => {
     });
   }, []);
 
-  const createStoryPreviewFromTemplate = useCallback(
-    async (template) => {
+  const createStoryPreview = useCallback(
+    async (dashboardStory) => {
       dispatch({
-        type: STORY_ACTION_TYPES.CREATING_STORY_PREVIEW_FROM_TEMPLATE,
+        type: STORY_ACTION_TYPES.CREATING_STORY_PREVIEW,
         payload: true,
       });
 
       try {
-        const { createdBy, pages, title, excerpt } = template;
+        const {
+          author,
+          createdBy,
+          created,
+          modified,
+          pages,
+          password,
+          title,
+          excerpt,
+          status,
+        } = dashboardStory;
 
         const storyProps = await getStoryPropsToSave({
           story: {
-            status: 'auto-draft',
+            status: status || 'auto-draft',
             title: title,
-            author: 1,
+            author: author || 1,
             slug: title,
-            date: Date.now().toString(),
-            modified: Date.now().toString(),
+            date: created ? created.format() : Date.now().toString(),
+            modified: modified ? modified.format() : Date.now().toString(),
```

Comment:
Are there times where we don't get dates from the data?

Reply:
Actually yes, the dates are empty for templates right now. `created` and `modified` exist as moment objects for stories.